Travel Scheduler/TravelSchedulerHelper.m

```diff
@@ -49,7 +49,7 @@ void setupGRonImagewithTaps(UITapGestureRecognizer *tgr, UIImageView *imageView,
 
 #pragma mark - Date method helpers
 
-NSDate* getNextDate(NSDate *date, int offset) {
+NSDate *getNextDate(NSDate *date, int offset) {
```

Comment:
Wait, is the asterisk supposed to be next to the method name? I sort of thought that it was connected to the NSDate because it's a pointer for NSDate? Honestly have no idea though.

Reply:
@aliu917 The asterisk should always be attached to whatever is on the right. In almost all cases because it is resolving/referencing that variable name, but also because of style.
Always `type *var_name`.
So Franklin's changes are correct!